CMakeLists.txt

```diff
@@ -129,11 +138,13 @@ if (CMAKE_WANT_MATERIALX_BUILD)
 endif()
 include(cmake/usd.cmake)
 
-if(${MAYA_APP_VERSION} STRLESS "2019" AND CMAKE_WANT_UFE_BUILD)
-    set(CMAKE_WANT_UFE_BUILD OFF)
-    message(AUTHOR_WARNING "========================================================================================= \
-                            UFE is not available in Maya${MAYA_MAJOR_VERSION}. Maya2019 and later are supported only. \
-                            ========================================================================================= ")
+if(DEFINED MAYA_APP_VERSION)
```

Comment:
If we don't call `find_package(Maya)` above then `MAYA_APP_VERSION` is not defined.